test/demo-logger-using-pvc/logger.py

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/python3
+
+import logging
+import socket
+import time
+import sys
+
+LOG_DIR = "/var/log/demoLogger/"
+HOSTNAME = socket.gethostname()
+LOG_FILE = LOG_DIR + HOSTNAME + ".log"
+
+def initialize_logger():
+    log = logging.getLogger("demo-logger")
+    log.setLevel(level=logging.DEBUG)
```

Comment:
log level should be configurable even, can we read this from the sample config file

Reply:
https://github.com/Seagate/cortx-prvsnr/pull/6134#discussion_r709463848
Can be addressed but will add unnecessary dependencies in the demo.

Reply:
thats fine

Reply:
added the comment. 👍